tools/pepquery2/pepquery2.xml

```diff
@@ -0,0 +1,809 @@
+                <param name="task_type" value="novel"/>
+            </conditional>
+            <section name="req_inputs">
+                <conditional name="input_type">
+                    <param name="input_type_selector" value="peptide"/>
+                    <conditional name="multiple">
+                        <param name="peptide_input_selector" value="single" />
+                        <param name="input" value="LVVVGADGVGK,AHSSMVGVNLPQK"/>
+                    </conditional>
+                </conditional>
+                <conditional name="db_type">
+                    <param name="db_type_selector" value="download" />
+                    <param name="db_id" value="gencode:human"/>
+                </conditional>
+                <conditional name="ms_dataset">
+                    <param name="ms_dataset_type" value="PepQueryDB"/>
+                    <param name="dataset" value="CPTAC_LUAD_Discovery_Study_Proteome_PDC000153" />
+                </conditional>
+                <param name="indexType" value="1"/>
+            </section>
+            <param name="parameter_set" value=""/>
+            <section name="modifications">
+                 <param name="fixed_mod" value="1"/>
+                 <param name="var_mod" value="2"/>
+                 <param name="max_mods" value="3"/>
+                 <param name="unmodified" value="True"/>
+                 <param name="aa" value="False"/>
+            </section>
+            <section name="digestion">
+                <param name="enzyme" value="1"/>
+                <param name="max_missed_cleavages" value="2"/>
+            </section>
+            <section name="ms_params">
+                <section name="tolerance_params">
+                    <param name="precursor_tolerance" value="10"/>
+                    <param name="precursor_unit" value="ppm"/>
+                    <param name="tolerance" value="0.6"/>
+                </section>
+                <section name="search">
+                    <param name="frag_method" value="1"/>
+                    <param name="scoring_method" value="1"/>
+                    <param name="extra_score_validation" value="False"/>
+                    <param name="min_charge" value="2"/>
+                    <param name="max_charge" value="3"/>
+                    <param name="min_peaks" value="10"/>
+                    <param name="isotope_error" value="0"/>
+                    <param name="min_score" value="12"/>
+                    <param name="min_length" value="7"/>
+                    <param name="max_length" value="45"/>
+                    <param name="num_random_peptides" value="1000"/>
+                </section>
+            </section>
+            <output name="psm_rank_txt">
+                <assert_contents>
+                    <has_text text="LVVVGADGVGK" />
+                    <not_has_text text="AHSSMVGVNLPQK" />
+                    <has_text text="02CPTAC_LUAD_W_BI_20180518_KR_f15:25149:2" />
```

Comment:
It could be useful to check for the number of columns in those outputs. This is a general comment as you are dealing with column headers etc.